site/content/en/docs/Reference/gameserver.md

```diff
@@ -190,20 +190,22 @@ Game Servers are created through Kubernetes API (either directly or through a [F
 
 [`GameServer.Status`][gss] has two fields which reflect the network address of the `GameServer`: `address` and `addresses`.
 The `address` field is a policy-based choice of "primary address" that will work for many use cases,
-and will always be one of the `addresses`. The `addresses` field contains every address in the [`Node.Status.addresses`][addresses],
+and will always be one of the `addresses`. The `addresses` field contains every address in the [`Node.Status.addresses`][addresses] and [`Pod.Status.podIPs`][podIPs] (to allow a direct pod access),
 representing all known ways to reach the `GameServer` over the network.
 
 To choose `address` from `addresses`, [Agones looks for the following address types][addressFunc], in highest to lowest priorty:
 * `ExternalDNS`
 * `ExternalIP`
 * `InternalDNS`
 * `InternalIP`
+* `PodIP`
```

Comment:
Since you didn't modify the `address` function in pkg/gameservers/gameservers.go, I don't think this line is true. However, to avoid a roundtrip across timezones and over a weekend, I may just go ahead and edit this, since the rest LGTM.

Reply:
(fixed in https://github.com/googleforgames/agones/pull/3764/commits/a671f1b91e511d13a5366f780a4175fbe081ed94)